gradle.properties

```diff
@@ -20,7 +20,7 @@ group=org.apache.kafka
 #  - tests/kafkatest/__init__.py
 #  - tests/kafkatest/version.py (variable DEV_VERSION)
 #  - kafka-merge-pr.py
-version=5.4.0-ccs-SNAPSHOT
+version=5.4.1-ccs-SNAPSHOT
```

Comment:
The comment above this line mentions a few other files. Did you check if they need to be updated?